cypress/e2e/with-users/machines/details.spec.ts

```diff
@@ -39,7 +39,7 @@ context("Machine details", () => {
     cy.findByRole("columnheader", { name: /subnet/i }).should("not.exist");
   });
 
-  it.only("displays machine commissioning details", () => {
```

Comment:
When I merged the eslint configs I might have neglected to include the `.only` rule for these tests.